File: library/src/matrix_energy/lib/content/MEBlocks.java
package matrix_energy.lib.content;

import matrix_energy.lib.world.blocks.environment.MEOreBlock;
import mindustry.type.Category;
import mindustry.type.ItemStack;
import mindustry.world.Block;

/**
 * @author dg
 */
public class MEBlocks {
    public static Block
        oreIron;
    public static void load(){
        oreIron = new MEOreBlock("ore-iron",MEItems.ironOreItems){{
            requirements(Category.distribution, ItemStack.with(MEItems.iron, 100));
        }};

    }
}

File: library/src/matrix_energy/lib/content/MEBlocks.kt
package matrix_energy.lib.content

import matrix_energy.lib.world.blocks.environment.MEOreBlock
import mindustry.type.Category
import mindustry.type.ItemStack
import mindustry.world.Block
import mindustry.world.blocks.environment.Floor

/**
 * @author dg
 */
object MEBlocks {
    var oreIron: Block? = null
    var floor0: Floor? = null

    @JvmStatic
    fun load() {
        oreIron = MEOreBlock("ore-iron", MEItems.ironOreItem10).apply {
            requirements(Category.distribution, ItemStack.with(MEItems.Fe, 100))

        }
        floor0 = Floor("floor0-test")
    }
}
